src/components/Banner/index.tsx

```diff
@@ -13,17 +13,11 @@ interface BannerProps {
  * Simple component that returns the NYPL Design System's Hero component but
  * with other necessary components
  */
-const Banner = ({
-  fullImageSrc = defaultImageSrc,
-  heroDescription = defaultDescription,
-}: BannerProps) => {
+const Banner = ({ fullImageSrc = defaultImageSrc }: BannerProps) => {
   const { t } = useTranslation("common");
   const h1Heading = (
     <Heading id="hero-banner" level="one" text={t("banner.title")} />
   );
-  const subHeaderText = (
```

Comment:
Why was this update necessary?

Reply:
It wasn't being used anywhere, and the linter flagged it, so I removed it